tests/test.sh

```diff
@@ -294,13 +328,21 @@ install_helm
 SECONDS=0
 # Wait for the test to complete
 sleep 120 # Wait for 2 minutes in order to give enough time to heapster pod to gather relevant stats
+$SRC_EXPORT
```

Comment:
Is this always correct .... you mix receiver /  generator with external and internal ... this is confusing .... the names are important for the understanding ....

Reply:
Yes, it is! SRC_EXPORT is the export that grants access to the cluster on which the generators reside (i.e. the traffic source) and vice versa.
I will add a comment to make things a little bit clearer.